utils/frame/benchmarking-cli/src/command.rs

```diff
@@ -176,7 +177,10 @@ impl BenchmarkCmd {
 			.filter(|item| pallet.is_empty() || pallet == &b"*"[..] || pallet == &item.pallet[..])
 			.for_each(|item| {
 				for benchmark in &item.benchmarks {
-					if extrinsic.is_empty() || extrinsic == &b"*"[..] || extrinsic == benchmark.name
+					let benchmark_name = &benchmark.name;
+					if extrinsic.is_empty() ||
+						extrinsic.contains(&&b"*"[..]) ||
```

Comment:
What did you think while adding the `contains("*")` here?

Reply:
So, incase `-- extrinsic *` is being used in executing the benchmark which is as it is before.

Reply:
But then you can just leave it as is with `extrinsic == "*"`, right?   
Otherwise it supports `ext1, ext2, *` which does not make so much sense IMHO.

Reply:
Oh... you know the extrinsic is a vector type already, so, i am not sure i will be able to do an `==` on the `"*"` string

Reply:
Currently you shadow the `extrinsic` string with a vector, if you name the vector `extrinsics` then this would not happen.

Reply:
I think i dig what you really mean, i will work on this and fix. Thanks

Reply:
made the fix. @ggwpez